templates/heatapi/config/heatapi-config.json

```diff
@@ -0,0 +1,43 @@
+{
+  "command": "/usr/sbin/httpd -DFOREGROUND",
+  "config_files": [
+    {
+      "source": "/var/lib/config-data/merged/heat.conf",
+      "dest": "/etc/heat/heat.conf",
+      "merge": false,
+      "preserve_properties": true,
+      "perm": "0644"
+    },
+    {
+      "source": "/var/lib/config-data/merged/heat-api-httpd.conf",
+      "dest": "/etc/httpd/conf/httpd.conf",
+      "merge": false,
+      "preserve_properties": true,
+      "perm": "0644"
+    },
+    {
+      "source": "/usr/bin/heat-wsgi-api",
+      "dest": "/var/www/cgi-bin/heat/heat_api",
+      "merge": false,
+      "preserve_properties": true,
+      "perm": "0644"
+    }
```

Comment:
probably we can use https://review.opendev.org/c/openstack/tripleo-common/+/874526 ?

Reply:
When we build the container to begin with you mean?

Reply:
Yes that is correct.

Giving it another though I guess we can use /usr/bin/ script instead of copying it to /var/lib/cgi-bin. The current symlink comes from what was done by puppet.

See https://github.com/openstack-k8s-operators/nova-operator/blob/master/templates/novaapi/config/httpd.conf#L42

Reply:
Since selinux is disabled in the container, we can probably fairly easily use the script from /usr/bin. I'll change the implementation and push a new commit to this PR before we merge.